applications/GeoMechanicsApplication/tests/cpp_tests/test_GeoMechanicsNewtonRaphsonErosionProcessStrategy.cpp

```diff
@@ -34,9 +34,14 @@ KRATOS_TEST_CASE_IN_SUITE(ErosionProcessStrategy, KratosGeoMechanicsIntegrationS
     auto projectFile = "ProjectParameters.json";
 
     auto execute = KratosExecute();
-    int status = execute.ExecuteFlowAnalysis(workingDirectory, projectFile, 3, 4, 0.1, "PorousDomain.Left_head",
-                                             &flow_stubs::emptyLog, &flow_stubs::emptyProgress,
-                                             &flow_stubs::emptyLog, &flow_stubs::emptyCancel);
+    Kratos::KratosExecute::CriticalHeadInfo critical_head_info(3, 4, 0.1);
+    Kratos::KratosExecute::CallBackFunctions call_back_functions(&flow_stubs::emptyLog,
+                                                                 &flow_stubs::emptyProgress,
+                                                                 &flow_stubs::emptyLog,
+                                                                 &flow_stubs::emptyCancel);
```

Comment:
These two structs can be const (same for the other ones in this file and in the other test file)
```suggestion
    const Kratos::KratosExecute::CriticalHeadInfo critical_head_info(3, 4, 0.1);
    const Kratos::KratosExecute::CallBackFunctions call_back_functions(&flow_stubs::emptyLog,
                                                                 &flow_stubs::emptyProgress,
                                                                 &flow_stubs::emptyLog,
                                                                 &flow_stubs::emptyCancel);
```